Bottom: 49a92ea799263e1c73be12282bcae6017c83b474
Top:    489d7194584daf06422d76eb0f23ba7ddf97ddf8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-19 20:41:35 +0000

Disable perturbation attempt during intersection


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 9948072..d0793d7 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1158,6 +1158,7 @@ seg_in_seg_arc_line (struct info *i, struct seg *s1, struct seg *s2)
 
   printf ("  Intersect count is %i\n", cnt);
 
+#if 0
   if (cnt == 0)
     {
       printf ("Trying with a perturbation\n");
@@ -1172,6 +1173,10 @@ seg_in_seg_arc_line (struct info *i, struct seg *s1, struct seg *s2)
 
       printf ("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! <<<<<<\n");
     }
+#endif
+
+  if (cnt == 0)
+    return 0;
 
   if (cnt == 2)
     {
